packages/api-server/src/base/types/api.ts

```diff
@@ -0,0 +1,70 @@
+import { Hash, HexNumber, HexString } from "@ckb-lumos/base";
+
+export interface EthTransaction {
```

Comment:
right now we can stay this way. later maybe  we can remove the Eth prefix by seperating eth and godwoken type through different namespace.